tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    value: "example"
+  - name: substrate-name
+    value: "YOUR_SUBSTRATE_NAME"
+  podTemplate:
```

Comment:
Is this necessary?

Reply:
This parameter is needed for the subnet selector. We can learn the subnet selector from Karpenter provisioner in the script but that requires read permission to the provisioner CRD. It will be easier to enter the name here.

Will rename this accordingly.

Reply:
This parameter is no longer needed.